config.yml

```
@@ -159,5 +159,36 @@ rules:
       - confidence check
       - coherence check
     severity: warning
+
+  - name: abort
+    terms:
+      - abort
+    alternatives:
+      - force quit
+      - halt
+      - close
+
```

"Abort" is quite a common term that may break CI in many places.
The default severity is `error`. Should we set it to `info`?

```suggestion
    severity: info
```

---

Hmm. Using `--exit-1-on-failure` doesn't really help even when severity is `info`.

---

I understand the reaction to downgrading the severity of a common term. But the entire purpose of inclusive naming is to strongly encourage a change of behaviour and use of common terms. 

For example, the term "master" was also a very common term which has been tackled via a great deal of effort. I believe, it's perfectly reasonable to expect the term "abort" to be replaced with "Force stopped" or any of [the alternatives listed](https://inclusivenaming.org/word-lists/tier-1/abort/).

I would suggest you set inclusive GitHub a actions to not required. Therefore not boxing development but acts as a warning until we have 100% coverage.

---

I would be cautious setting any GH action to be non-required. As they tend to get ignored as we get adjusted to them often or always failing. The point of GH actions is to block work in order to incite change.

You could make it a required check and then adjust the action using a project specific config to suit your needs.